frontend/src/pages/modelServing/screens/global/InferenceServiceListView.tsx

```diff
@@ -106,7 +106,6 @@ const InferenceServiceListView: React.FC<InferenceServiceListViewProps> = ({
         clearFilters={resetFilters}
         servingRuntimes={servingRuntimes}
         inferenceServices={filteredInferenceServices.slice(pageSize * (page - 1), pageSize * page)}
```

Comment:
If you are going to do the pagination in the table, I think you don't need to do the slice here, right? you can just simply pass all the `filteredInferenceService` into `InferenceServiceTable`.